### DISCUSSION TITLE
Add new Label: Select 'Color' by name of an existing label

### DISCUSSION THREAD

**Discussion by HenrikBengtsson** [community/community#40562] [Projects and Issues]
### Select Topic Area

Product Feedback

### Wish

When creating a new label for issues, you often want to clone the color of an existing label, e.g. all bug-related labels should be `#d73a4a`.   To make cloning colors easier, please consider updating the 'Color' field to also support selecting the color based on an existing label.

To distinguish label searching from hex-code entering, maybe require an explicit `#` for the latter, or make it possible to remove it for the former.

![image](https://user-images.githubusercontent.com/1616850/205108872-96167103-4665-4709-81e5-47eb4cde84b1.png)

**Comment by ossanna16** [comment]
Hi there @HenrikBengtsson and welcome to our community! Thank you for the feedback!
